wandb/sdk/internal/stats.py

```diff
@@ -150,7 +150,7 @@ def _thread_body(self) -> None:
                 time.sleep(0.1)
                 seconds += 0.1
                 if self._shutdown:
-                    self.flush()
+                    self.flush()  # type: ignore
```

Comment:
is this expected that you need a type ignore?

Reply:
yes, it's calling not-fully-typed functions. i might get to ++'ing our annotation coverage later in the week/next week, will fix then. my main task here was to make it pass on the latest mypy version. :)